Dockerfile

```diff
@@ -18,15 +19,17 @@ COPY cmd/oam-kubernetes-runtime/main.go main.go
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o controller main.go
 
-FROM ghcr.io/oracle/oraclelinux:8-slim
-USER root
 RUN groupadd -g 65532 nonroot && \
-    useradd -g 65532 -u 65532 -d /home/nonroot -m nonroot && \
-    microdnf update -y && \
-    microdnf update -y openssl-libs glibc-common && \
```

Comment:
what about the openssl-libs and glibc-common updates? are these not needed?

Reply:
I'm not sure why those libraries were explicitly updated. The binary is statically linked so as far as I can tell, those libraries are not used.